wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/ConfigurationMessageCell/Components/ConversationSystemMessageCell.swift

```diff
@@ -91,6 +91,43 @@ extension ConversationStartedSystemMessageCell {
 
 }
 
+class ConversationEncryptionInfoCell: ConversationIconBasedCell,ConversationMessageCell {
+    struct Configuration {}
```

Comment:
Why not use this Configuration to set two labels? We can then reuse this cell in the future. Also, the `configureSubviews()` method will not contain label values.

Reply:
I think it's unlikely we'll have any other cell looking exactly like this (with exactly two labels and image view at level of second label). We have a lot of generic code thats set to be used in multiple places that isn't. And it make reading code harder

Reply:
done